core/utils/kagome_db_editor.cpp

```diff
@@ -2,12 +2,19 @@
 #include <fstream>
 #include <thread>
 
+#include <backward.hpp>
+#undef TRUE
+#undef FALSE
```

Comment:
Hm... Why?

Reply:
it clashes in definition in some other lib. sorry, don't remember. actually won't compile without it